utils/src/com/goide/GoEnvironmentUtil.java

```diff
@@ -13,15 +12,22 @@
 
 public class GoEnvironmentUtil {
   public static final String GO_EXECUTABLE_NAME = "go";
-  
+  public static final String GAE_EXECUTABLE_NAME = "goapp";
+
   private GoEnvironmentUtil() {
   }
 
-  @NotNull
+  @Nullable
```

Comment:
>  I'd prefer for the clients to have a way to check

The method return File, not path, so clients have plenty ways to check it.

> it's not nice not having any feedback for the user in case something went wrong as to why it happened.

You'll get feedback from GeneralCommandLine. getExecutableForSdk method is just 'getter', it doesn't give any guarantees about correctness of returning file and I believe it shouldn't just because it cannot know how client will use the result.

Anyway, as I see you're solving at least two problems in this commit. The first one is about looking for GAE-executable. And the second one is not really clear for me. Let's separate this commit it two, I'll merge the first one and we'll discuss the second.